packages/acgtk/acgtk.1.5.2/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "sylvain.pogodalla@inria.fr"
+
+build: [
+  ["dune" "subst"] {pinned}
+# remove the -p to also build the local libraries: conflict with the
+# fact that some libraries are also part of the acgtkLib package
+#  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "--profile=release" "-j" jobs]
+]
+
+install: ["dune" "install"]
+
+depends: [
+  "ocaml" { >= "4.05.0" }
+  "dune" { >= "1.0" }
```

Comment:
```suggestion
  "dune" { >= "1.4" }
```
The dune-project file requires 1.4 (https://gitlab.inria.fr/ACG/dev/ACGtk/-/blob/master/dune-project#L1), can you upstream the change?
For the rest looks good

Reply:
Done (hopefully!).